index.Rmd

```diff
@@ -106,4 +106,4 @@ You should be aware of the University of Toronto Code of Behaviour on Academic M
 | Reproducible workflow | Project, Individual | Nov 06     | 8     |
 | Final project work    | Project, Group      | Dec 9 – 20 | 32    |
 
-_There will be a penalty of 5% per day (including week-ends) for late submissions._
+Assessments will be distributed and submitted in the R-markdown format via Blackboard or GitHub. _There will be a penalty of 5% per day (including week-ends) for late submissions._
```

Comment:
I know I'm nit picking here, but it should be `R Markdown`. Sorry, I can sometimes be a perfectionist...